src/graph.jl

```diff
@@ -163,6 +163,61 @@ function all_node_properties(table::Shapefile.Table)::Array{Dict{String, Any}}
 end
 
 
+function get_attribute_by_key(node_attributes::Array,
+                              column_name::String,
+                              process_value::Function=identity)::Array
+    """ Returns an array whose values correspond to the value of an attribute
+        for each node.
+
+        Arguments:
+
+        node_attributes :   An array of Dict{String, Any}, where each dictionary
+                            represents a mapping from attribute names to values
+                            for a particular node.
+        column_name     :   The name of the attribute (i.e., the key of the
+                            attribute in the dictionaries)
+        process_value   :   An optional argument that processes the raw value
+    """
+    return [process_value(n[column_name]) for n in node_attributes]
```

Comment:
Does this helper function make things significantly shorter where it's used?

Reply:
i think so! i mean, it also makes it much easier for us to write functions that transform process the raw value of somehow (i.e., doing type conversions)